GitUI/CommandsDialogs/SettingsDialog/Pages/BuildServerIntegrationSettingsPage.cs

```diff
@@ -70,6 +70,8 @@ protected override void SettingsToPage()
 
                     BuildServerType.SelectedItem = buildServerSettings.ServerName ?? _noneItem.Text;
                     ActivateBuildServerSettingsControl();
+
+                    base.SettingsToPage();
                 });
```

Comment:
❔
```suggestion
                    base.SettingsToPage();
                }).FileAndForget();
```

Reply:
No idea :) Seems to be working as is... 
This may require further investigation, but I can't allocate any time for this.

Reply:
This is a generic issues which should be dealt with separately.